concert/experiments/imaging.py

```diff
@@ -736,30 +750,8 @@ async def _finish_radios(self):
         )
         self._finished = True
 
-    async def _take_radios(self):
-        """
-        Generator for projection images
-
-        First the :py:meth:`_prepare_radios()` is called. Then the camera is set to trigger_source
-        'SOFTWARE'.
-        Then num_projections * num_tomograms images are recorded. For each image i the
-        tomography_motor is moved to angular_range/num_projections * i.
-        The vertical motor is moved to vertical_shift_per_tomogram / num_projections * i`.
-        """
-        try:
-            num_projections = int(await self.get_num_projections() * await self.get_num_tomograms())
-            await self._prepare_radios()
-            await self._camera.set_trigger_source("SOFTWARE")
-            async with self._camera.recording():
-                for i in range(num_projections):
-                    await self._prepare_frame(i)
-                    await self._camera.trigger()
-                    yield await self._camera.grab()
-        finally:
-            await self._finish_radios()
-
```

Comment:
both `_take_radios` slipped into this commit I guess.

Reply:
Do you like this commit msg more?